client/FxAccountClient.js

```diff
@@ -791,14 +813,20 @@ define([
    * @param {String} email
    * @param {String} oldPassword
    * @param {String} newPassword
+   * @param {String} sessionToken
+   * @param {Object} [options={}] Options
+   *   @param {Boolean} [options.keys]
+   *   If `true`, calls the API with `?keys=true` to get the keyFetchToken
    * @return {Promise} A promise that will be fulfilled with JSON `xhr.responseText` of the request
    */
-  FxAccountClient.prototype.passwordChange = function(email, oldPassword, newPassword) {
+  FxAccountClient.prototype.passwordChange = function(email, oldPassword, newPassword, sessionToken, options) {
     var self = this;
+    options = options || {};
 
     required(email, 'email');
     required(oldPassword, 'old password');
     required(newPassword, 'new password');
+    required(sessionToken, 'sessionToken');
```

Comment:
That seems fair. Currently, if `sessionToken` is not passed, [an unverified session](https://github.com/mozilla/fxa-auth-server/blob/verify-token-full/lib/routes/password.js#L162) is return for password change finish. Does it still make sense to do that?